components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -264,7 +264,7 @@ - (void)encodeWithCoder:(NSCoder *)aCoder {
 }
 
 - (void)setTitleFont:(UIFont *)titleFont {
-  _titleFont = [titleFont fontWithSize:kTitleFontSize];
+  _titleFont = [UIFont fontWithName:titleFont.fontName size:kTitleFontSize];
```

Comment:
Please add  a comment along the lines of:
// -(UIFont)fontWithSize: does not maintain the weight of the original font so we are 
// using +(UIFont)fontWithName:size:

Reply:
done.